### PR TITLE
Avoid potential overflows

### PR DESCRIPTION
**Pull request by blt**
### What does this PR do?

There is a bug in lading where gauge values for smaps_rollup, smaps
come through in the EiB range. The changes here are not a fix but they
don't hurt either and they're a good idea.

### CODE REVIEW

**Comment by blt** [comment]
* **#1138** <a href="https://app.graphite.dev/github/pr/DataDog/lading/1138?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/DataDog/lading/1138?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: lading/src/observer/linux/procfs/memory/smaps_rollup.rs [hunk 1]
```diff
@@ -66,12 +68,16 @@ pub(crate) async fn poll(
...
+            "rss" => aggr.rss = aggr.rss.saturating_add(value_bytes),
+            "pss" => aggr.pss = aggr.pss.saturating_add(value_bytes),
             _ => { /* ignore other fields */ }
         }
         let metric_name = format!("smaps_rollup.{field}");
-        gauge!(metric_name, labels).set(value_kb as f64);
+        info!(
+            "line: {line}, value_bytes: {value_bytes}, bytes: {bytes}",
+            bytes = value_bytes as f64
+        );
```
Comment (GeorgeHahn) [inline lading/src/observer/linux/procfs/memory/smaps_rollup.rs:79]: This looks noisy - can it be removed or reduced in level?

Comment (blt) [reply lading/src/observer/linux/procfs/memory/smaps_rollup.rs:79]: Oops that's a debugging line I did not mean to leave. Good catch.

**Review by GeorgeHahn — APPROVED** [review]

**Comment by blt** [comment]
### Merge activity

* **Dec 10, 8:46 PM EST**: A user merged this pull request with [Graphite](https://app.graphite.dev/github/pr/DataDog/lading/1138).

**Pull request merged**